erpnext/hr/doctype/salary_slip/salary_slip.py

```diff
@@ -815,6 +819,23 @@ def update_status(self, salary_slip=None):
 				timesheet.set_status()
 				timesheet.save()
 
+	def update_loans(self, status):
+		for loan in self.get_loan_details():
+			doc = frappe.get_doc("Loan", loan.name)
+			if status == "Submit":
```

Comment:
Can be based on docstatus, no need to pass args

Reply:
done